app/javascript/src/LoginPage/loginForms/RequestPassWordForm.jsx

```diff
@@ -17,56 +22,53 @@ type Props = {
 }
 
 type State = {
-  emailAddress: string,
-  isValidEmail: ?boolean
+  email: string,
+  validation: {
+    email: ?boolean
+  }
 }
 
 class RequestPasswordForm extends React.Component<Props, State> {
-  constructor (props: Props) {
-    super(props)
-    this.state = {
-      emailAddress: '',
-      isValidEmail: undefined
+  state = {
+    email: '',
+    validation: {
+      email: undefined
```

Comment:
No need to initialize as `undefined`, it can be simply `validation: {}`. On the other hand, I think it would be good to initialize all validations to `false` (not only in this form but all of them).

Reply:
I disagree, if we initialise all fields to false, the form will display error messages by default.  Like this:
![errors](https://user-images.githubusercontent.com/13486237/61691768-8e5aea80-ad2c-11e9-8c62-7c8a84e92cb8.png)

IMO error messages should be shown only if the user has done an actual error, not before.
WDYT @thomasmaas  ?

Reply:
Yes of course. Showing a form with error messages before user interaction would be super aggressive.